artemis-distribution/src/main/resources/licenses/bin/LICENSE

```diff
@@ -286,6 +286,12 @@ For HdrHistogram:
 This product bundles HdrHistogram, which is available under the
 "2-clause BSD" license. For details, see licences/LICENSE-hdrhistogram.txt.
 
+==============================================================================
+For JLine:
+==============================================================================
+This product bundles JLine, which is available under the
+"3-clause BSD" license. For details, see licences/LICENSE-jline.txt.
```

Comment:
Referenced file still needs added.

Reply:
It was added here but apparently .txt is on the gitignore and I missed adding it. thank you.